openshift/kustomize/cron/base/config-map.yaml

```diff
@@ -16,47 +16,38 @@ metadata:
     created-by: jeremy.foster
 data:
   entrypoint.sh: |-
-    #!/bin/sh
+    #!/bin/bash
```

Comment:
busybox doesn't have `bash`, but `sh` doesn't have a good way to use custom functions.